ui/src/components/Investor/InvestorOrders.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react'
+import { Button, Card, Header, Form, Message } from 'semantic-ui-react'
+
+import { useLedger, useStreamQuery } from '@daml/react'
+import { Order, OrderRequest } from '@daml.js/da-marketplace/lib/Marketplace/Trading'
+
+import { ExchangeIcon, OrdersIcon } from '../../icons/Icons'
+import { unwrapDamlTuple, wrapDamlTuple } from '../common/Tuple'
+import Page from '../common/Page'
+
+import './InvestorOrders.css'
+
+type Props = {
+    sideNav: React.ReactElement;
+    onLogout: () => void;
+}
+
+const InvestorOrders: React.FC<Props> = ({ sideNav, onLogout }) => {
+    const allOrders = useStreamQuery(Order).contracts;
+    const allOrderRequests = useStreamQuery(OrderRequest).contracts;
+
+    return (
+        <Page
+            sideNav={sideNav}
+            menuTitle={<><OrdersIcon/>Orders</>}
+            onLogout={onLogout}
+        >
+            <div className='investor-orders'>
+                <Header as='h4'>Requested Orders</Header>
+                { allOrderRequests.map(or => <OrderCard key={or.contractId} order={or.payload.order}/>)}
+
+                <Header as='h4'>Open Orders</Header>
+                { allOrders.map(o => <OpenOrderCard key={o.contractId} order={o.payload}/>)}
+            </div>
+        </Page>
+    )
+}
+
+type OrderProps = {
+    order: Order;
+}
+
+const OrderCard: React.FC<OrderProps> = ({ children, order }) => {
+    const base = unwrapDamlTuple(order.pair)[0].label;
+    const label = order.isBid ? `Bought ${base}` : `Sold ${base}`;
```

Comment:
since these are not trades, let's just name them "buy" or "sell". Not "bought" or "sold". (I'll also let jennifer know in case her designs threw you off)